framework/src/st_things/things_stack/src/common/framework/things_security_manager.c

```diff
@@ -226,10 +226,32 @@ static OCStackResult seckey_setup(const char *filename, OicSecKey_t *key, OicEnc
 		return OC_STACK_ERROR;
 	}
 
+	if (encoding == OIC_ENCODING_UNKNOW) {
```

Comment:
not unknown? Is unknow right?

Reply:
@sunghan-chang 'UNKNOW' is right. It is fixed in iotivity release code.

Reply:
What's the meaning of "fixed"?

Reply:
@sunghan-chang It is using just like that. I can not modify it as the iotivity defined it,